### ISSUE TITLE
Supported Archtecture

### ISSUE THREAD

**Issue by M1Sports20** [kabouzeid/nvim-lspinstall#95]
I use lspinstall on multiple architectures, obviously lspinstall only runs on x86. It would be nice if it didn't install x86 binaries if I am on a different architectures.  
For now I just have, I am fine if this is the long term fix:
```
if system("uname -m") == "x86_64l\n"
 do lsp stuff.
endif
```

**Comment by kabouzeid** [comment]
Some servers already support ARM. I don't have a device to test on. Feel free to make PRs for this.

**Comment by kabouzeid** [comment]
See terraform and rust for examples

**Comment by M1Sports20** [comment]
Yes, looks like terraform and rust distribute binaries.  I don't think the ones I look at do.  For example: clangd.  Darn

**Comment by kabouzeid** [comment]
You could open an issue there, asking if they would be willing to also provide arm binaries in their releases. IIRC they are pretty responsive.

https://github.com/clangd/clangd/releases

**Comment by M1Sports20** [comment]
Yeah.  There is an open ticket already: https://github.com/clangd/clangd/issues/514
Guess we can wait until this is complete.
